packages/core/src/components/table/table-body-row-expandable/table-body-row-expandable.scss

```
@@ -69,6 +69,11 @@
       padding: 16px 16px 16px 66px;
       color: var(--tds-table-color);
     }
+
+    .tds-table__cell-expand-visible {
+      color: var(--tds-table-color);
+      padding: 16px 16px 16px 66px;
+    }
```

```suggestion

    .tds-table__cell-expand-overflow-visible {
      overflow: visible;
      color: var(--tds-table-color);
      padding: 16px 16px 16px 66px;
    }
    
      .tds-table__cell-expand-overflow-hidden {
      overflow: hidden;
      color: var(--tds-table-color);
      padding: 16px 16px 16px 66px;
    }
```
To make it more clear what we change. It might be confusing as we have expand-visible, in future we might think this is something related to make the expand row visible and not control of overflow property.